server/portal/libs/agave/operations.py

```diff
@@ -245,42 +248,38 @@ def move(client, src_system, src_path, dest_system, dest_path, file_name=None):
         file_name = src_path.strip('/').split('/')[-1]
 
     dest_path_full = os.path.join(dest_path.strip('/'), file_name)
-    src_path_full = urllib.parse.quote(src_path)
 
     # Handle attempt to move a file into its current path.
-    if src_system == dest_system and src_path_full == dest_path_full:
-        return {'system': src_system, 'path': src_path_full, 'name': file_name}
+    if src_system == dest_system and src_path == dest_path_full:
+        return {'system': src_system, 'path': src_path, 'name': file_name}
 
     try:
         # list the directory and check if file_name exists
-        file_listing = client.files.list(systemId=dest_system, filePath=dest_path)
+        file_listing = client.files.listFiles(systemId=dest_system, path=dest_path)
         file_name = increment_file_name(listing=file_listing, file_name=file_name)
-    except HTTPError as err:
-        if err.response.status_code != 404:
-            raise
-
-    full_dest_path = os.path.join(dest_path.strip('/'), file_name)
+        dest_path_full = os.path.join(dest_path.strip('/'), file_name)
+    except BaseTapyException:
```

Comment:
should the try/except be removed in the 3 spots?

Reply:
Yeah I guess it can be removed since we aren't doing anything extra really after catching the exception. Is that why you were thinking of removing it?

Reply:
yes, exactly. 👍    and if solution in follow-on-issue requires a try/except block then we can re-add.